bears/general/InvalidLinkBear.py

```diff
@@ -1,5 +1,7 @@
 import re
 import requests
+from collections import defaultdict
```

Comment:
Sorry for that, hadn't had time for self review.. I'll be doing it now. Thanks for looking out!